packages/grafana-data/src/types/datasource.ts

```diff
@@ -284,6 +298,7 @@ export interface ExploreQueryFieldProps<
 > extends QueryEditorProps<DSType, TQuery, TOptions> {
   history: any[];
   onBlur?: () => void;
+  absoluteRange?: AbsoluteTimeRange;
```

Comment:
Why is this added here? 

cc @ryantxu

Reply:
An `absoluteRange` prop is passed to `QueryField` components at some points (see QueryRow.tsx, line 168), which causes an error in typescript 3.7 since `absoluteRange` wasn't a member of the props interface.

Full error:
```
Type '{ datasource: DataSourceApi<DataQuery, DataSourceJsonData>; query: DataQuery; history: HistoryItem<DataQuery>[]; ... 5 more ...; absoluteRange: AbsoluteTimeRange; }' is not assignable to type 'IntrinsicAttributes & ExploreQueryFieldProps<DataSourceApi<DataQuery, DataSourceJsonData>, DataQuery, DataSourceJsonData> & { ...; }'.
  Property 'absoluteRange' does not exist on type 'IntrinsicAttributes & ExploreQueryFieldProps<DataSourceApi<DataQuery, DataSourceJsonData>, DataQuery, DataSourceJsonData> & { ...; }'.ts(2322)
```